.husky/pre-commit

```diff
@@ -5,7 +5,7 @@ yarn pretty-quick --staged
 
 BRANCH_NAME=$(git branch --show-current)
 
-if [ "$BRANCH_NAME" = "main" ]; then
+if [ "$BRANCH_NAME" = "maina" ]; then
```

Comment:
😕

Reply:
Sorry, that file should have not been committed